calibration.py

```diff
@@ -0,0 +1,55 @@
+__author__ = 'Patrick B. Grinaway'
+
+import openeye.oechem as oechem
+import gaff2xml
+import simtk.openmm as openmm
+import simtk.openmm.app as app
+import simtk.unit as units
+import joblib
+
+class Calibrator(object):
+
+    """
+    Given a ligand system, generate new sets of charges
+    representing fluorination states, as well as biases for the expanded ensemble simulation in implicit solvent.
```

Comment:
Get rid of `fluorination states`.

This is presumably the scheme that generates initial guesses for the log weights?  Might be useful to update the docs for this.